### ISSUE TITLE
Can we release a patch version that supports the latest version of Pagy?

### ISSUE THREAD

**Issue by cpjmcquillan** [DFE-Digital/govuk-components#389]
Equally, if we're wanting to test the change more thoroughly, I can 
push that change to a branch and point our application at it for testing.

**Comment by peteryates** [comment]
I _think_ this should be solved now by 4.0.0a1?

**Comment by cpjmcquillan** [comment]
Thanks @peteryates.

Just gave it a go and looks like we'd need a rc for `govuk_design_system_formbuilder` too due to this conflict.

```
In snapshot (Gemfile.lock):
    html-attributes-utils (>= 0.9.2)

  In Gemfile:
    govuk-components (= 4.0.0a1) was resolved to 4.0.0a1, which depends on
      html-attributes-utils (>= 1.0.0, ~> 1.0.0)

    govuk_design_system_formbuilder was resolved to 3.3.0, which depends on
      html-attributes-utils (~> 0.9, >= 0.9.2)
```

**Comment by peteryates** [comment]
@cpjmcquillan done, it's building now!
